### PR TITLE
Update docs in preparation for repo rename

### PR DESCRIPTION
**Pull request by baronfel**
I did a cleanliness pass over the docs, changing references to the new expected repo name (dotnet/sdk-container-builds), and scrubbing up some of the duplicated content. I still want to touch the README.

### CODE REVIEW

File: .github/workflows/publish-package.yml [hunk 1]
```diff
@@ -24,4 +24,4 @@ jobs:
     - name: Release
       run: dotnet build --configuration release
     - name: Publish
-      run: dotnet nuget push "Microsoft.NET.Build.Containers/bin/Release/Microsoft.NET.Build.Containers.*.nupkg" --api-key ${{ secrets.FEED_UPLOAD_TOKEN }} --source "https://nuget.pkg.github.com/rainersigwald/index.json"
+      run: dotnet nuget push "Microsoft.NET.Build.Containers/bin/Release/Microsoft.NET.Build.Containers.*.nupkg" --api-key ${{ secrets.FEED_UPLOAD_TOKEN }} --source "https://nuget.pkg.github.com/dotnet/index.json"
```
Comment (baronfel) [inline .github/workflows/publish-package.yml:27]: this feed will likely change post-move I'm guessing

File: Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj [hunk 2]
```diff
@@ -24,8 +24,8 @@
     <Description>Tasks and targets to natively publish .NET applications as containers.</Description>
     <Copyright></Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageProjectUrl>https://github.com/rainersigwald/containers</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/rainersigwald/containers</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/dotnet/sdk-container-builds</PackageProjectUrl>
```
Comment (baronfel) [inline Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj:27]: turns out that the github package feed needs these urls to be in alignment :D

File: DEVGUIDE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+## Using the prerelease package from this repository
+
+The easiest way to use the prerelease package is to follow [these GitHub docs](https://docs.github.com/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry).
+
+You'll want to use a nuget.config file to
+
+* define a source for this repository
+* define credentials (username and Personal Access Token) to use when accessing this source
+
+Then, you can use `dotnet add package Microsoft.NET.Build.Containers prerelease` to get the latest version.
```
Comment (rainersigwald) [inline DEVGUIDE.md:14]:
```suggestion
Then, you can use `dotnet add package Microsoft.NET.Build.Containers -prerelease` to get the latest version.
```

?

File: README.md [hunk 4]
```diff
@@ -1,26 +1,29 @@
...
+From there, you can simply `dotnet build` the repository and be good to go!
 
-You don't "build" New Repo, however, this will be meaningful for many other projects.
 
 ## Contributing
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) for information on contributing to this project.
 
-This project has adopted the code of conduct defined by the [Contributor Covenant](http://contributor-covenant.org/) 
+This project has adopted the code of conduct defined by the [Contributor Covenant](http://contributor-covenant.org/).
```
Comment (rainersigwald) [inline README.md:26]:
```suggestion
This project has adopted the code of conduct defined by the [Contributor Covenant](http://contributor-covenant.org/)
```

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**